common/workunit/workunit.cpp

```diff
@@ -6452,22 +6464,37 @@ class CDaliWorkUnitFactory : public CWorkUnitFactory, implements IDaliClientShut
                 case WUStateAborting:
                     if (agentSessionStopped)
                     {
-                        reportAbnormalTermination(wuid, ret, agent);
+                        reportAbnormalTermination(wuid, ret, agentSessionID, "Agent");
+                        return ret;
+                    }
+                    if (engineSessionStopped)
+                    {
+                        reportAbnormalTermination(wuid, ret, engineSessionStopped, "Engine");
```

Comment:
Should be passing engineSessionID

Reply:
whoops(2) - yes, would only affect tracing but.. now fixed.